app/source/js/firebase-helper.js

```diff
@@ -8,19 +8,30 @@ firebase.initializeApp(firebaseInit);
 const database = firebase.database();
 
 const onNotification = callback => {
-  database
-    .ref('notifications')
+  const reference = database.ref('notifications');
+
+  reference
     .orderByChild('time')
     .limitToFirst(30)
     .on('value', snapshot => {
       callback(firebaseToArray(snapshot));
     });
+
+  return () => {
```

Comment:
Since `reference.off` is already a function, is it necessary to wrap it?

Reply:
Yeah, it did not work to return `reference.off` and call it from the component.

Reply:
Maybe `NOTE` it then? :)

Reply:
Hmm... maybe. IMO it was better than to start `bind`ing.